packages/webchannel-wrapper/src/index.d.ts

```diff
@@ -44,13 +86,22 @@ export interface WebChannelOptions {
   fastHandshake?: boolean;
   disableRedac?: boolean;
   clientProfile?: string;
-  internalChannelParams?: { [key: string]: boolean | number };
+  internalChannelParams?: {
+    forwardChannelRequestTimeoutMs?: number;
+  };
   xmlHttpFactory?: unknown;
   requestRefreshThresholds?: { [key: string]: number };
 }
 
+export interface WebChannel {
+  open(): void;
+  close(): void;
+  listen<T>(type: string, cb: (param: T) => void): void;
```

Comment:
I still don't really understand this usage of generics.  Is there even any practical difference between:
```
function foo(x: any)
// and
function foo<T>(x: T)
```
It seems like it's a way of telling TypeScript to let you pass anything.  The generic way looks more type-safe, but I don't think it is?

It seems like the correct type here is `unknown` and we should explicitly cast to something else if we need to.

Reply:
No - there is no difference, but one looks safer! I changed this to `unknown` and modified the consumer.